implementations/elixir/ockam/ockam_kafka/lib/kafka.ex

```diff
@@ -87,6 +89,64 @@ defmodule Ockam.Kafka do
     end
   end
 
+  def get_topics(options) do
+    brod_options = brod_options(options)
+
+    case :brod.get_metadata(brod_options.endpoints, :all, brod_options.client_config) do
+      {:ok, %{topic_metadata: topics}} ->
+        Enum.map(topics, fn topic ->
+          {Map.get(topic, :topic), Enum.count(Map.get(topic, :partition_metadata))}
+        end)
+
+      other ->
+        other
+    end
+  end
+
+  def resolve_offset(topic, partition, time, options) do
+    brod_options = brod_options(options)
+
+    :brod.resolve_offset(
+      brod_options.endpoints,
+      topic,
+      partition,
+      time,
+      brod_options.client_config
+    )
+  end
+
+  def fetch(topic, partition, offset, options) do
+    brod_options = brod_options(options)
+
+    :brod.fetch(
+      brod_options.endpoints,
+      topic,
+      partition,
+      offset,
+      20,
+      1,
```

Comment:
I would prefer to have the numbers used into functions defined as constant within the module otherwise it is difficult to understand why these numbers are used